packages/nextjs-contentful-renderer/jest.config.js

```diff
@@ -0,0 +1,188 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Not for this pr but at some point we might want to create a base `jest.config.js` file and extend it from other packages

Reply:
https://impulsumvc.atlassian.net/browse/COM-574